### DISCUSSION TITLE
How do you become qualified for the leaderboard?

### DISCUSSION THREAD

**Discussion by unrealapex** [monkeytypegame/monkeytype#1203] [Questions] [answered]
I was wondering how you become qualified for the leaderboard; is there a average wpm or accuracy you need?

**Comment by frogfrogfrogfrogfrog** [comment, marked as answer]
I'm pretty sure you have to get enough wpm in order to get onto the leaderboard. For example if the slowest typist on the leaderboard is typing at 140 wpm you'd have to type faster than that to get onto the leaderboard. I've never been able to type fast enough to get onto the leaderboard myself so I don't know if there are any other requirements.
